doc/lua_api.txt

```diff
@@ -3652,8 +3765,50 @@ Definition tables
 * deprecated, yet still supported field names:
     * `image` (name)
 
-### Tile animation definition
-* `{type="vertical_frames", aspect_w=16, aspect_h=16, length=3.0}`
+### Animation definition
+
+#### Node animation, particle and particle spawners
+* `{	type="vertical_frames", 
+	aspect_w=16, 
+    --  ^ specify width of a picture in pixels. 
+	aspect_h=16, 
+    --  ^ specify height of a frame in pixels. 
+	length=3.0
+    --  ^ specify full loop length. 
+	first_frame = 0, -- <- only for particles, use
+	min_first_frame = 0, -- <- for particle spawners
+	max_first_frame = 0,
+	loop_animation = true,  -- <- only for particles and particle spawners
+    --  specify if animation should start from beginning after last frame.
+}`
+
+#### Particle and particle spawners only
+* `{	
+	type="2d_animation_sheet", -- <- only for particles and particle spawners
+	vertical_frame_num = 1,
+	horizontal_frame_num = 1,
+    --  ^ specify amount of frames in texture. 
+    --	Can be used both for animation or for texture transform 
+    --	together with first_frame variable.
+    --	A animation texture separated on equal parts of frames,
+    --	by horizontal and vertical numbers. For example with 
+    --	vertical_frame_num = 4 and horizontal_frame_num = 3 we got
+    --	4*3 = 12 frames in total. Animation sequence start from
+    --	left top frame and go on to the right until reach end of
+    --	row. Next row also start from left frame.
+	first_frame = 0, -- <- only for particles, use
+	min_first_frame = 0, -- <- for particle spawners
+	max_first_frame = 0,
+    --  ^ specify first frame to start animation.
+	frame_length = -1,
```

Comment:
Hm i guess because we have 'first frame' and min/max first frame, no longer is the frame length decided by 'length / number of frames'. Do we need 'length' at all then, could animation speed be decided by 'frame length'?